col/src/main/java/vct/col/ast/Node.scala

```diff
@@ -729,6 +737,7 @@ final case class JavaNamedType[G](names: Seq[(String, Option[Seq[Type[G]]])])(im
   var ref: Option[JavaTypeNameTarget[G]] = None
 }
 final case class JavaTClass[G](ref: Ref[G, JavaClassOrInterface[G]], typeArgs: Seq[Type[G]])(implicit val o: Origin = DiagnosticOrigin) extends JavaType[G] with JavaTClassImpl[G]
+final case class TJavaString[G]()(implicit val o: Origin = DiagnosticOrigin) extends JavaType[G] with TJavaStringImpl[G]
```

Comment:
Could you change out this type (and the names of the two coercions) such that we have a type for any SpecialDecl? Something like:

```
final case class TSpecial[G](kind: SpecialDecl[G])
final case class CoerceSpecialToConcreteClass(...)
final case class CoerceConcreteClasstoSpecial(...)
```

That way we can have a list of SpecialDecl's, but we don't have to add new similar types and coercions every time.